lib/within.js

```diff
@@ -21,18 +21,24 @@ function within(context, fn) {
       if (helpers[helper]._withinBegin) recorder.add('start within block', () => helpers[helper]._withinBegin(context));
     });
 
-    fn();
+    try{
+      fn();
+    }
+    catch(e){
+      //need suggestion how to report error with respect to codeceptJS
```

Comment:
I would like to report it to user if callback fails on exception. Not sure how to do it in codecept way, hoping for guidance here.